LICENSE-MIT

```diff
@@ -1,6 +1,6 @@
-MIT License
+The MIT License (MIT)
 
-Copyright (c) 2020 Electric Coin Company
+Copyright (c) 2017-2021 The Electric Coin Company
```

Comment:
While bringing in the dual-license change from our other codebases, the change to re-introduce the original author's notice was removed. This should be:
```suggestion
Copyright (c) 2017 CikeQiu
Copyright (c) 2020-2021 The Electric Coin Company
```